src/group/managed_group/mod.rs

```diff
@@ -53,10 +56,9 @@ use ser::*;
 /// Changes to the group state are dispatched as events through callback
 /// functions (see [`ManagedGroupCallbacks`]).
 #[derive(Debug)]
-#[cfg_attr(test, derive(PartialEq))]
 pub struct ManagedGroup<'a> {
-    // CredentialBundle used to sign messages
-    credential_bundle: &'a CredentialBundle,
+    // Reference to the KeyStore to obtain private key material.
+    key_store: &'a KeyStore,
```

Comment:
This implies that the `KeyStore` is owned by the client (however that looks like) and there's only one key store? This should also be clarified in the `KeyStore` module docs.

Reply:
Added a sentence for clarification in the KeyStore docs.